code/modules/reagents/newchem/other.dm

```diff
@@ -120,7 +120,7 @@ datum/reagent/colorful_reagent
 	mix_message = "The substance flashes multiple colors and emits the smell of a pocket protector."
 
 datum/reagent/colorful_reagent/reaction_mob(var/mob/living/simple_animal/M, var/method=TOUCH, var/volume)
-    if(M)
+    if(M && istype(M))
         M.color = pick(random_color_list)
```

Comment:
@Iamgoofball 
Like Fox said, its `reaction_mob` and `on_mob_life` were removed in #681 (you even commented on the removed `on_mob_life`), the `reaction_mob` was re-added for corgis in #816 (but a missing type check made it color everything), and the type check was added in here.

I wasn't aware of the murdering it caused, though. I thought the problem was it screwing up HUDs by also coloring them.

@CrAzYPiLoT-SS13 
You might have been testing the Set Default Language verb, which always worked just fine. The issue was in the Check Known Languages window, which you can *also* set your default language in. The names of the languages were breaking the HTML. In hindsight, tweaking the HTML wasn't the best way to fix this, but it should be fine unless we get a language containing quotation marks.